clients/drcachesim/tools/record_filter_launcher.cpp

```diff
@@ -32,6 +32,12 @@
 
 /* Standalone record filter tool launcher for file traces. */
 
+#ifdef WINDOWS
+#    define NOMINMAX // Avoid windows.h messing up std::max.
+#    define UNICODE
```

Comment:
So we need both UNICODE defs? Maybe add a comment describing why they're required (e.g. I'm not familiar with some Windows-specific things like this)

Reply:
Yes, IIRC one is Windows headers and one is C headers.  Right, confirmed: https://stackoverflow.com/questions/7953025/why-both-unicode-and-unicode

Added a comment.